Dockerfile

```diff
@@ -101,6 +103,12 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install --no-instal
     texlive-latex-recommended \
     texlive-science
 
+# spead2_net_raw is needed so that qualification tests can use ibverbs
+RUN wget https://raw.githubusercontent.com/ska-sa/spead2/master/src/spead2_net_raw.cpp && \
```

Comment:
I'd suggest specifying the output path. At the moment it's wherever a previous build stage happens to leave WORKDIR, which could cause some spooky action at a distance if we make changes elsewhere in the file.

Reply:
Makes sense.